Cargo.toml

```diff
@@ -13,7 +13,7 @@ keywords = ["cli", "console", "ratatui", "terminal", "tui"]
 derive_builder = "0.13.0"
 font8x8 = "0.3.1"
 itertools = "0.12.0"
-ratatui = "0.25.0"
+ratatui = "0.26.0"
```

Comment:
I think perhaps it's worth doing >= 0.25 for the lib and 0.26 for the examples

Reply:
Definitely yes!